src/components/Screen.tsx

```diff
@@ -125,6 +126,7 @@ export const InnerScreen = React.forwardRef<View, ScreenProps>(
         <DelayedFreeze freeze={freezeOnBlur && activityState === 0}>
           <AnimatedScreen
             {...props}
+            style={[style, { zIndex: undefined }]}
```

Comment:
I'd add a comment that we are handling the order of screens on the native side and changing it here causes issues, with the link to the issue.

Reply:
+1